File: baidu-hot-search/src/main/java/io/github/llmagentbuilder/tool/baiduhotsearch/BaiduHotSearchConfig.java
package io.github.llmagentbuilder.tool.baiduhotsearch;

public class BaiduHotSearchConfig {

  private int numberOfItems = 10;

  public int getNumberOfItems() {
    return numberOfItems;
  }

  public void setNumberOfItems(int numberOfItems) {
    this.numberOfItems = numberOfItems;
  }

  @Override
  public String toString() {
    return "BaiduHotSearchConfig{" +
        "numberOfItems=" + numberOfItems +
        '}';
  }
}

File: baidu-hot-search/src/main/kotlin/io/github/llmagentbuilder/tool/baiduhotsearch/BaiduHotSearchConfig.kt
package io.github.llmagentbuilder.tool.baiduhotsearch

data class BaiduHotSearchConfig(val numberOfItems: Int? = null)

data class BaiduHotSearchRequest(val numberOfItems: Int? = null)

data class BaiduHotSearchResponse(val results: List<String>? = null)